src/java/org/apache/cassandra/db/marshal/VectorType.java

```
@@ -174,8 +176,6 @@ public <V> V decomposeAsFloat(ValueAccessor<V> accessor, float[] value)
     {
         if (!(elementType instanceof FloatType))
             throw new IllegalStateException("Attempted to read as float, but element type is " + elementType.asCQL3Type());
-        if (value == null)
```

if this happens we NPE, so should we throw an error with a useful msg if we don't want to return `null`?

This change is also not consistent with `fromComparableBytes`